specs/tables.md

```diff
@@ -36,84 +36,87 @@ Proved by the state circuit.
-| $counter | $isWrite    | CallContext                | $callID  |                | ReversibleWriteCounter     |             | $value     | 0          | 0               |
-|          |             |                            |          |                |                            |             |            |            |                 |
-| $counter | $isWrite    | Stack                      | $callID  | $stackPointer  |                            |             | $value     | 0          | 0               |
-| $counter | $isWrite    | Memory                     | $callID  | $memoryAddress |                            |             | $value     | 0          | 0               |
-| $counter | $isWrite    | AccountStorage             | $txID    | $address       |                            | $storageKey | $value     | $valuePrev | $committedValue |
-|          |             |                            |          |                |                            |             |            |            |                 |
-| $counter | true        | TxLog                      | $txID    | $logID         | Address                    | 0           | $value     | 0          | 0               |
-| $counter | true        | TxLog                      | $txID    | $logID         | Topic                      | $topicIndex | $value     | 0          | 0               |
-| $counter | true        | TxLog                      | $txID    | $logID         | Data                       | $byteIndex  | $value     | 0          | 0               |
-| $counter | true        | TxLog                      | $txID    | $logID         | TopicLength                | 0           | $value     | 0          | 0               |
-| $counter | true        | TxLog                      | $txID    | $logID         | DataLength                 | 0           | $value     | 0          | 0               |
-|          |             |                            |          |                |                            |             |            |            |                 |
-| $counter | false       | TxReceipt                  | $txID    | 0              | PostStateOrStatus          | 0           | $value     | 0          | 0               |
-| $counter | false       | TxReceipt                  | $txID    | 0              | CumulativeGasUsed          | 0           | $value     | 0          | 0               |
-| $counter | false       | TxReceipt                  | $txID    | 0              | LogLength                  | 0           | $value     | 0          | 0               |
+Details:
+
+- **Address (key2)** is reserved for stack, memory, and account addresses.
+- **StorageKey (key4)** is reserved for RLC encoded values
+- **value, valuePrev**: variable size, depending on Tag (key0) and FieldTag (key3) where appropriate.
+- **(rw) counter**: 32 bits, starts at 1.
+- **txID**: 32 bits, starts at 1 (corresponds to `txIndex - 1`).
```

Comment:
`txIndex + 1` ? as I understand `txIndex` starts with zero

Reply:
It should be `txIndex + 1`! (I made the same mistake as in logID)

I checked and txID starts at 1. See:
https://github.com/privacy-scaling-explorations/zkevm-specs/blob/8952cf88101e6a6decd6396636e0982eea1931c8/src/zkevm_specs/evm/execution/begin_tx.py#L14-L16

Reply:
Fixed (the incorrect sign) in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/215/commits/4d4cc7e660d508d3dd1e9db84449519eab926edb